infrastructure/main.tf

```diff
@@ -209,6 +64,17 @@ locals {
   }
 }
 
+data "azurerm_key_vault_secret" "microservice_keys" {
```

Comment:
what does this do? get all the secrets?

Reply:
Yes, what happened here is I merged two `locals` sections and as a result this resource and the one below have been moved down a bit. They've already been here.